cypress/fixtures/tableByUserResponses.js

```diff
@@ -0,0 +1,58 @@
+export const USER_RESPONSES = {
+  'mock-member-id-1': [
+    {
+      qName: 'Fill In The Blanks',
+      fields: {
+        answer:
+          'Lorem <ipsum> dolor sit amet, consectetur adipiscing elit. <ips um> ut fermentum nulla, sed <suscipit> sem.',
+        date: 'Fri Jul 22 2022',
+        icon: 'CancelOutlinedIcon',
+      },
+    },
+    {
+      qName: 'What is a baby cat called?',
+      fields: {
+        answer: '90',
+        date: 'Fri Jul 22 2022',
+        icon: 'CancelOutlinedIcon',
+      },
+    },
+    {
+      qName: 'What is the capital of France?',
+      fields: {
+        answer: 'Paris',
+        date: 'Fri Jul 22 2022',
+        icon: 'CheckCircleOutlinedIcon',
+      },
+    },
+  ],
+  'mock-member-id-2': [
+    {
+      qName: 'How happy are you?',
+      fields: {
+        answer: '60',
+        date: 'Fri Jul 22 2022',
+        icon: 'CancelOutlinedIcon',
+      },
+    },
+    {
+      qName: 'What is the capital of France?',
```

Comment:
Was the property previously called `qName`? Isn't it duplicate string from the question's data? I would maybe refer to the question id instead.